templates/legal/index.html

```diff
@@ -23,76 +23,57 @@ <h1>Ubuntu legal</h1>
-            <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/b06cbaa6-picto-canonical-warmgrey.svg" alt="" />
-            <h3 class="p-heading-icon__title">
-              <a href="/legal/contributors">
-                Contributor agreement&nbsp;&rsaquo;
-              </a>
-            </h3>
-          </div>
-          <p>Details of the agreement you need to send to us before you begin contributing to projects at Canonical.</p>
-        </div>
+        <h3>
+          <a href="/legal/trademarks">Trademarks&nbsp;&rsaquo;</a>
+        </h3>
+        <p>A list of trademarks used by Canonical.</p>
       </div>
       <div class="col-6 p-card">
-        <div class="p-heading-icon">
-          <div class="p-heading-icon__header">
-            <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/2a8cf960-picto-startfirst-warmgrey.svg" alt="" />
-            <h3 class="p-heading-icon__title">
-              <a href="/legal/managed-services">
-                Managed Services terms&nbsp;&rsaquo;
-              </a>
-            </h3>
-          </div>
-          <p>A legal description of the services included in Bootstack and Managed Kubernetes.</p>
-        </div>
+        <h3>
+          <a href="/legal/contributors">Contributor agreement&nbsp;&rsaquo;</a>
+        </h3>
+        <p>Details of the agreement you need to send to us before you begin contributing to  projects at Canonical.</p>
       </div>
     </div>
   </div>
   <div class="row">
-    <div class="col-12 p-card">
-      <div class="p-heading-icon">
-        <div class="p-heading-icon__header">
-          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/0838729a-picto-bookmark-warmgrey.svg" alt="" />
-          <h3 class="p-heading-icon__title">
-            <a href="/legal/short-terms">
-              Short terms&nbsp;&rsaquo;
-            </a>
-          </h3>
-        </div>
-        <p>This agreement sets out Canonical&rsquo;s standard service terms for both Ubuntu Advantage and Managed Services customers.</p>
+    <div class="u-equal-height">
+      <div class="col-6 p-card">
+        <h3>
+          <a href="/legal/ubuntu-advantage">Ubuntu Advantage terms&nbsp;&rsaquo;</a>
+        </h3>
+        <p>A legal description of the services included in the Ubuntu Advantage support package.</p>
       </div>
+      <div class="col-6 p-card">
+        <h3>
+          <a href="/legal/managed-services">Managed Services terms&nbsp;&rsaquo;</a>
+        </h3>
+        <p>A legal description of the services included in Bootstack and Managed Kubernetes.</p>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12 p-card">
+      <h3>
+        <a href="/legal/service-terms">Service terms&nbsp;&rsaquo;</a>
+      </h3>
+      <p>This agreement sets out Canonical&rsquo;s standard service terms for both Ubuntu Advantage and Managed Services customers.</p>
     </div>
   </div>
 </div>
```

Comment:
The copydoc shows a four column footer after this?

Reply:
@pmahnke Does that ^ seem right to you? We don't have a 4 col footer anywhere else as far as I can see.